### ISSUE TITLE
Bug Reports. Input type date showing wrong dates that are first day of a month, changing the day 1 with 32 of the past month

### ISSUE THREAD

**Issue by rperdomoEM** [tabalinas/jsgrid#1390]
I'm using the 1.5.3 version.

the bug is simple:

internally, the date is saved as, for example, "2019-04-01" and then, in the table is loaded as "2019-03-32".

when I edit the row, it shows the correct date, it saves it well internally; but when I submit a day 1 of a month, it shows it as the day 32 of the past month.

when I filter with the first day of a month, it gives all them, but with the day 32 of the past month.

when I create a day 1 of a month, it saves it well internally, but shows the day 32 of the past month.

TL/DR: jsGrid saves the date well, but when it shows a day that is first of a month, it shows the day 32 of the past month.

**Comment by rperdomoEM** [comment]
sorry, my bad. there is no Date in jsGrid
